src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.regex.Pattern;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] checkEmpty = new String[0];
+        File file = new File(fileName);
+
+        if (file.length() == 0) {
```

Comment:
this check is redundant, if file is empty you will skip the loop

Reply:
done